lib/operations/command.rb

```diff
@@ -264,7 +265,7 @@ def possible(params = EMPTY_HASH, **context)
   # These 3 methods added for convenience. They return boolean result
   # instead of Operations::Result. True on success and false on failure.
   %i[callable allowed possible].each do |method|
-    define_method "#{method}?" do |**kwargs|
+    define_method :"#{method}?" do |**kwargs|
```

Comment:
What was the reason for this change? Is it rubocop?

Reply:
yes